depthai_helpers/calibration_utils.py

```diff
@@ -89,15 +88,15 @@ def polygon_from_image_name(image_name):
 class StereoCalibration(object):
     """Class to Calculate Calibration and Rectify a Stereo Camera."""
 
-    def __init__(self, traceLevel: int):
-        self.traceLevel = traceLevel
+    def __init__(self):
         """Class to Calculate Calibration and Rectify a Stereo Camera."""
 
-    def calibrate(self, board_config, filepath, square_size, mrk_size, squaresX, squaresY, camera_model, enable_disp_rectify):
+    def calibrate(self, board_config, filepath, square_size, mrk_size, squaresX, squaresY, camera_model, enable_disp_rectify, display_all):
```

Comment:
What is display_all ?

I would rather have traceLevel with some maxNumber which does displayall.

Issue with display all is it floods your terminal making it hard to debug.

Reply:
Setting the trace value to 10, if you want to display all the data in calibration.